README.md

```diff
@@ -125,6 +125,63 @@ async function go() {
 go();
 ```
 
+### Command-Line Runner
+
+> A new experimental command-line test runner is available. Its API _may_ change as we get feedback on it. Please try it out!
+
+After installing the package, the runner is available as the `vscode-test` CLI. Running it will look for a `.vscode-test.(js/json/mjs)` file relative to the current working directory. You can see the configuration [here](https://github.com/microsoft/vscode-test/blob/main/lib/cli-runner/config.ts). This may be as simple as:
+
+```js
+// .vscode-test.js
+module.exports = { files: 'out/test/**/*.test.js' };
```

Comment:
Does it have a standard test pattern? I think it would be nice if this could run without a configuration file.

Also it’s pretty common nowadays to add a JSDoc type annotation to JavaScript configuration files, or to expose a function `defineConfig` that does nothing, except provide type hints and documentation. I think many people would appreciate that.

For example: https://playwright.dev/docs/test-configuration, https://vitest.dev/config/

```js
const { defineConfig } = require('@vscode/test-electron')

module.exports = defineConfig({
  // …
})
```